machine.go

```diff
@@ -719,7 +719,11 @@ func (machine Machine) toNode(
 
 		KeepAlive:         true,
 		MachineAuthorized: !machine.isExpired(),
-		Capabilities:      []string{tailcfg.CapabilityFileSharing},
+		Capabilities: []string{
+			tailcfg.CapabilityFileSharing,
+			tailcfg.CapabilityAdmin,
```

Comment:
Not sure if `CapabilityAdmin` is needed 🤔

Reply:
This is one of the important open questions. If it isn’t present, the tailscale client won’t let you use `--ssh`. However, I don’t know what else it enables or if it should be included only for certain users.